Modules/OrgUnit/classes/User/class.ilOrgUnitUser.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace OrgUnit\User;
+
+use OrgUnit\Positions\ilOrgUnitPosition;
+
+class ilOrgUnitUser {
+
+	/**
+	 * @var self[]
+	 */
+	protected static $instances;
+	/**
+	 * @var int
+	 */
+	protected $user_id;
+	/**
+	 * @var string
+	 */
+	protected $login;
+	/**
+	 * @var string
+	 */
+	protected $email;
```

Comment:
Please keep in mind that a user can have a `second_email` (since ILIAS 5.3.x AFAIR).